nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestAnnotationService.java

```diff
@@ -456,4 +457,9 @@ public void testDeleteExternalAnnotation() {
 
     }
 
+    protected String getParentIdOdAnnotatedDoc(DocumentModel docToAnnotate) {
+        return session.getChildren(new IdRef(docToAnnotate.getId()),
+                CommentsConstants.COMMENTS_DIRECTORY_TYPE).get(0).getId();
```

Comment:
By design you could get the `CommentRoot` directly without having to do a query.